modules/installation-creating-gcp-iam-shared-vpc.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-iam-shared-vpc_{context}"]
+= Creating IAM roles in GCP
+
+You must create IAM roles in Google Cloud Platform (GCP) for your
+{product-title} cluster to use. One way to create these components is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your GCP
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and associated subnets in GCP.
+
+.Procedure
+
+. Copy the template from the
+*Deployment Manager template for IAM roles*
+section of this topic and save it as `03_iam.py` on your computer. This
+template describes the IAM roles that your cluster requires.
+
+. Export the following variables required by the resource definition. You must provide the same values that you used when you customized the `<installation_directory>/manifests/cloud-provider-config.yaml` file.
+.. Export the name of the host project control plane subnet:
++
+----
+$ export HOST_PROJECT_CONTROL_SUBNET=<control_plane_subnet>
```

Comment:
As the VPC network is created by google deployment template in the previous section, we actually know the name of subnet. So I suggest to change it to:
```javascript
$ export HOST_PROJECT_CONTROL_SUBNET="${INFRA_ID}-master-subnet"
```